terraform/hcl/applicable/req_sw_terraform_aws_alb_https_only.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_lb_listener" "vulnerable_example" {
+  load_balancer_arn = "arn:aws:iam::123456789012:user/johndoe"
+  port              = "443"
+  protocol          = "HTTP"
+  ssl_policy        = "ELBSecurityPolicy-2016-08"
+  certificate_arn   = "arn:aws:iam::187416307283:server-certificate/test_cert_rab3wuqwgja25ct3n4jdj2tzu4"
+
+  default_action {
+    type             = "forward"
+    target_group_arn = "arn:aws:iam::123456789012:user/johndoe"
+  }
+}
```

Comment:
[comment]: <> (FrogbotReview Iac 1 1 12 2 vulnerable_example /terraform/hcl/applicable/req_sw_terraform_aws_alb_https_only.tf)

### 🛠️ Infrastructure as Code Vulnerability
	
Severity: ![](https://raw.githubusercontent.com/jfrog/frogbot/master/resources/v2/applicableHighSeverity.png)<br>    high

Finding: `AWS Load balancer using insecure communications`

#### 👇 Details

<details>
<summary> <b>Full description</b> </summary>
<br>
Resources `aws_lb_listener` and `aws_alb_listener` should set `protocol = "HTTPS"` (default is `"HTTP"`)

Vulnerable example -
```
resource "aws_lb_listener" "vulnerable_example" {
  protocol          = "HTTP"
}
```

Secure example -
```
resource "aws_lb_listener" "secure_example" {
  protocol          = "HTTPS"
}
```

</details>

[[🐸 JFrog Frogbot]](https://github.com/jfrog/frogbot#readme)